plugins/filesystem/local/adapter/adapter.php

```diff
@@ -12,6 +12,9 @@
 JLoader::import('joomla.filesystem.file');
 JLoader::import('joomla.filesystem.folder');
 
+require_once 'filenotfoundexception.php';
```

Comment:
These files need to stay in com_media as they are the same for all future cloud plugins.